### ISSUE TITLE
Old time Clossys

### ISSUE THREAD

**Issue by L0RI-ORIGIN** [TunisianEagles/SocialBox#47]
hi, i need password for Clossys6.3.2, please help me, tnk

**Comment by alejandro79907-lang** [comment]
there are new clossys btw

**Comment by alejandro79907-lang** [comment]
use aloha

**Comment by TheShadowsHackers** [comment]
> use aloha

Where New Clossys

**Comment by alejandro79907-lang** [comment]
I can't tell you, I'll just say that you can't get it in a GitHub repository or anything like that, it's through contacts, not everything is found by searching on Google

**Comment by alejandro79907-lang** [comment]
we're on clossys 9.xx.xx so you get the idea

**Comment by TheShadowsHackers** [comment]
> I can't tell you, I'll just say that you can't get it in a GitHub repository or anything like that, it's through contacts, not everything is found by searching on Google

What is the Use of the latest version, give their Contact, If it Works I pay them

**Comment by alejandro79907-lang** [comment]
Shit, I hate people who only do it for the thrill and really don’t want to make an effort and just want to pay. Well, here’s the contact, his name is Prometeus, you can find him on Discord.good luck

**Comment by alejandro79907-lang** [comment]
Wow! A day has passed, how has it been for you? He's a somewhat serious guy, I don't think he would have accepted, but I know a drug addict who will surely tell you in exchange for money. His name is J17, also known as Lotus. You can contact him on Discord or ask Prometeus about his Discord.Be respectful with him please, it's not nice to get into trouble with him

**Comment by QuantumTeriz** [comment]
> > I can't tell you, I'll just say that you can't get it in a GitHub repository or anything like that, it's through contacts, not everything is found by searching on Google
> 
> What is the Use of the latest version, give their Contact, If it Works I pay them

Don't believe this guy, he's a fcking liar.
The Clossys 9.8 worked for Anonet, yes, it worked, it doesn't work anymore, it's obsolete along with their browser and network.
Don't believe the larpers and script kid/lamers.

**Comment by QuantumTeriz** [comment]
> Wow! A day has passed, how has it been for you? He's a somewhat serious guy, I don't think he would have accepted, but I know a drug addict who will surely tell you in exchange for money. His name is J17, also known as Lotus. You can contact him on Discord or ask Prometeus about his Discord.Be respectful with him please, it's not nice to get into trouble with him

Man, you don't even know J, is a nice guy, but don't bother him with that, because people like you are stressed with everything related to the Marianas website, plus you don't even know what the clossys 9.8 was in and you will never know because it's an obsolete clossys, and its not "real marianas", stop liar, stop larp. 😊

**Comment by tuntun676767yooo-oss** [comment]
Brother Clos network exists, what's more, I'm going to give it to you now, I'm going to give you a method which is the mapping in Tor, also called Horus Config

**Comment by tuntun676767yooo-oss** [comment]
Brother, the worst part is that you're right, I didn't know anything back then.

**Comment by tuntun676767yooo-oss** [comment]
And by the way, J17 is serious when he's not high.

**Comment by QuantumTeriz** [comment]
> > use aloha
> 
> Where New Clossys

And aloha is here on github, lol.

**Comment by tuntun676767yooo-oss** [comment]
I'm really sorry, you learn as you go and I didn't know what I was saying

**Comment by tuntun676767yooo-oss** [comment]
I don't know how Aloha Browser works, Truth only gave me the Horus config file. Anyway, the curious one gets the worst of it.

**Comment by tuntun676767yooo-oss** [comment]
Are you from OTB or TCN?

**Comment by tuntun676767yooo-oss** [comment]
<img width="1080" height="1440" alt="Image" src="https://github.com/user-attachments/assets/1059ac76-722d-431b-8862-ca94208675ba" />  look vroskidaroskipodoroskinegaroskitaparoskiteloroskitlemanoski

**Comment by tuntun676767yooo-oss** [comment]
Made by truth

**Comment by QuantumTeriz** [comment]
> And by the way, J17 is serious when he's not high.

Yeah, you don't fcking know Jesse, you only know him from Marianas web and that video from clossys, you've never talked to him, you've never really been his friend

**Comment by tuntun676767yooo-oss** [comment]
I only said that he's serious when he's not high; he's a good person.

**Comment by QuantumTeriz** [comment]
> <img alt="Image" width="1080" height="1440" src="https://private-user-images.githubusercontent.com/312825714/632129838-1059ac76-722d-431b-8862-ca94208675ba.jpg?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3ODU5ODk0MjYsIm5iZiI6MTc4NTk4OTEyNiwicGF0aCI6Ii8zMTI4MjU3MTQvNjMyMTI5ODM4LTEwNTlhYzc2LTcyMmQtNDMxYi04ODYyLWNhOTQyMDg2NzViYS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwODA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDgwNlQwNDA1MjZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05YjE0Zjc5ODVmMjQ1MjRhZWVlNzI5Y2M2YWVkNTcwNDY5YmY0MGNmMTFmZjE1MTA4ZDViYjk3ZGJkOWI5OGNhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZyZXNwb25zZS1jb250ZW50LXR5cGU9aW1hZ2UlMkZqcGVnIn0.4J42apwVy_qiNGa4VHLiJDztTQYcRQJrDJBkCjTDIUg"> look vroskidaroskipodoroskinegaroskitaparoskiteloroskitlemanoski

Oh nice, I like this page, It has a lot of information from forums, it's good that someone is collecting that, dark forums

**Comment by tuntun676767yooo-oss** [comment]
Would you like me to send you the Horus config?

**Comment by QuantumTeriz** [comment]
> I only said that he's serious when he's not high; he's a good person.

Yes, I know what you're talking about, I'm friends with j, and he's not serious all the time, he's not even serious most of the time, he's a chill guy, and please stop seeing your friends as a tool, that won't lead you to anything good

**Comment by tuntun676767yooo-oss** [comment]
Alraight

**Comment by QuantumTeriz** [comment]
> Would you like me to send you the Horus config?

What is horus?

**Comment by tuntun676767yooo-oss** [comment]
A configuration is one of the few ways to access Closed Network

**Comment by QuantumTeriz** [comment]
> A configuration is one of the few ways to access Closed Network

Thats always interesting, well the clos network has always been somewhat ambiguous, from DarkFantasy, dfn, deep vpn and a lot more projects, I'd like to see "horus"

**Comment by tuntun676767yooo-oss** [comment]
I know what you're trying to do, hahaha

**Comment by tuntun676767yooo-oss** [comment]
Horus is mapping in the Tor browser; in the Tor file you add text that says mapaddres And you place the Clos link and then you place an onion link

**Comment by QuantumTeriz** [comment]
> I know what you're trying to do, hahaha

What I try to do? Lol

**Comment by QuantumTeriz** [comment]
> Horus is mapping in the Tor browser; in the Tor file you add text that says mapaddres And you place the Clos link and then you place an onion link

Well, yeah it's a good way to create a private network using the tor client, a modified torrc file where the mapaddress translates the domain, a synthetic dns

**Comment by tuntun676767yooo-oss** [comment]
Are you from OTB or TCN?
